website/docs/r/cdn_domain_v1.html.markdown

```diff
@@ -45,11 +46,12 @@ The following arguments are supported:
 
 The `sources` block supports:
```

Comment:
The "Changing this parameter will ..." should be removed for `sources` as well at L41.

Reply:
Thanks for your comment, I have removed this description.